client/tests/e2e/snapshots/utils.js

```diff
@@ -0,0 +1,113 @@
+const prettier = require('prettier');
+
+const snapshotRoutes = [
+  '/',
+  '/signup-instructions',
+  '/account/settings',
+  '/test-queue',
+  '/reports',
+  '/candidate-review',
+  '/data-management',
+  '/404',
+  '/test-plan-report/1',
+  '/test-review/8',
+  '/run/2',
+  '/data-management/meter',
+  '/candidate-test-plan/24/1'
+];
+
+async function cleanAndNormalizeSnapshot(page) {
+  // Remove all <style> and <link rel="stylesheet"> elements
+  await page.evaluate(() => {
+    const stylesToRemove = document.querySelectorAll(
+      'style, link[rel="stylesheet"]'
+    );
+    stylesToRemove.forEach(el => el.remove());
+  });
+
+  // Remove elements with Date or status content that might differ between local and CI
+  await page.evaluate(() => {
+    function removeElements(selector) {
+      const elements = document.querySelectorAll(selector);
+      elements.forEach(el => {
+        const text = el.textContent.trim();
+        if (
+          text.includes('Target') ||
+          text.includes('Days Past') ||
+          text.includes('Review Completed') ||
+          text.includes('Target Completion Date') ||
+          text.includes('Ready for Review') ||
+          text.includes('Review in Progress') ||
+          text.match(/\w{3} \d{1,2}, \d{4}/)
+        ) {
+          el.remove();
+        }
+      });
+    }
+
+    removeElements('span, p.review-text, span.review-complete, div.info-label');
+    removeElements('td > i');
+    removeElements('.ready-for-review, .in-progress');
+  });
+
+  const content = await page.content();
+
+  // Strip out "Previously Viewed" badges and other dynamic content
+  let cleanedContent = content.replace(
+    /<span[^>]*class="[^"]*viewed-badge[^"]*"[^>]*>Previously Viewed<\/span>/g,
+    ''
+  );
+
+  // Remove href and version number on data management page
+  // Addresses inconsistent links and date -> version number
```

Comment:
A think a TODO should be included for ones like this so we can explore and remove this cleaner when necessary, explaining what the issue is.

There certainly shouldn't be a mismatch with the version dates being displayed no matter the environment.